package.json

```diff
@@ -24,10 +24,16 @@
     "publish": "babel-node -- scripts/server.js",
     "test": "npm run test:unit && npm run test:integration && npm run test:quick",
     "test:unit": "cp test/unit/config.js node_modules/config.webpack-node.js && NODE_PATH=$NODE_PATH:. BABEL_ENV=production mocha-webpack --webpack-config webpack/test.babel.js --require test/unit/setup.js \"test/unit/*.spec.js\"",
-    "test:integration": "cd test && bundle exec rspec spec/integration_spec.rb && cd ..",
+    "test:integration": "cd test && rspec spec/integration_spec.rb  --tag integration && cd ..",
```

Comment:
do we still have the `:integration` tag?

Reply:
I think it is useful for running everything once locally

Reply:
I don't see the `integration` tag in the code

Reply:
I erased the tag